praw/__init__.py

```diff
@@ -887,43 +887,35 @@ def get_front_page(self, *args, **kwargs):
         return self.get_content(self.config['reddit_url'], *args, **kwargs)
 
     @decorators.restrict_access(scope='read')
-    def get_info(self, url=None, thing_id=None, limit=None):
-        """Look up existing items by thing_id (fullname) or url.
+    def get_info(self, url=None, thing_id=None, *args, **kwargs):
+        """Return a get_content generator for things to look up.
 
-        :param url: The url to lookup.
+        :param url: A url to lookup.
         :param thing_id: A single thing_id, or a list of thing_ids. A thing_id
             can be any one of Comment (``t1_``), Link (``t3_``), or Subreddit
             (``t5_``) to lookup by fullname.
-        :param limit: The maximum number of Submissions to return when looking
-            up by url. When None, uses account default settings.
-        :returns: When a single ``thing_id`` is provided, return the
-            corresponding thing object, or ``None`` if not found. When a list
-            of ``thing_id``s or a ``url`` is provided return a list of thing
-            objects (up to ``limit``). ``None`` is returned if any one of the
```

Comment:
@bboe IIRC, as long as there is a valid id it still works, it just yields nothing if *all* ids are invalid.